### ISSUE TITLE
generate updated list of canonical transcripts

### ISSUE THREAD

**Issue by Chris-lang478** [ntm/grexome-TIMC-Secondary#3]
@ntm 
Nicolas,
Thanks again for your great work.
I was trying to generate updated list of canonical transcripts follow your scripts. However, I met ERROR 2003 (HY000): Can't connect to MySQL server on 'ensembldb.ensembl.org:3306' (110).
I am new to Linux system, I have tried google for the solution, but I just can't figure it out. And I don't have root access.
Could you offer more details for 'Just run it from eg nicofree'? Or could you offer the listCanonicalTranscripts_22017.tsv.gz?
Thanks again for your help!

Best regards, 
Chris

**Comment by ntm** [comment]
Hi @Chris-lang478 ,
I suspect the issue here is that listCanonicalTranscripts_MYSQL.sh connects to the public Ensembl MySQL server, on port TCP/3306. Unfortunately, outgoing traffic to this port is sometimes blocked by your institutional firewall - this is the case where I work, and being root on your system wouldn't help. As a workaround, I run that script from my home computer (called nicofree): this works fine, my ISP at home doesn't filter outgoing traffic.
You could do the same, but I'm just attaching the latest version for you: 
[listCanonicalTranscripts_220717.tsv.gz](https://github.com/ntm/grexome-TIMC-Secondary/files/9741723/listCanonicalTranscripts_220717.tsv.gz)
Cheers,
Nicolas

**Comment by ntm** [comment]
I see in another issue that my resolution solved this, closing
